mavros_extras/src/plugins/odom.cpp

```diff
@@ -341,46 +232,30 @@ class OdometryPlugin : public plugin::PluginBase {
 		Matrix6d r_vel = Matrix6d::Zero();	//!< Zero initialized velocity 6-D Covariance matrix. WRT child_frame_id
 
 		mavlink::common::msg::ODOMETRY msg {};
+		msg.frame_id = utils::enum_value(MAV_FRAME::VISION_NED);
+		msg.child_frame_id = utils::enum_value(MAV_FRAME::BODY_FRD);
 
 		/**
-		 * Position and orientation are in the same frame as frame_id.
-		 * For a matter of simplicity, and given the existent MAV_FRAME
-		 * enum values, the default frame_id will be a local frame of
-		 * reference, so the pose is WRT a local frame.
+		 * Position parsing from odometry's parent frame to "vision_ned" frame.
```

Comment:
Comments lies, see above. Let's use LOCAL_FRD instead of vision_ned to be consistent